tests/drivers/build_all/pwm/boards/it82xx2_evb.overlay

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2024 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "../../../../../dts/riscv/ite/it8801-mfd-map.dtsi"
```

Comment:
Is the relative path necessary?  Does `#include <ite/it8801-mfd-map.dtsi>` work instead?

Reply:
When using the PWM of IT8801, we must select the alternate pin of the GPIO. The` mfdctrl = <&pwm7_gp20_default>` in dts is defined in "it8801-mfd-map.dtsi", so we need to Include this file.

Fixed to `#include <ite/it8801-mfd-map.dtsi>`